recipes/pykml/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pykml" %}
+{% set version = "0.1.0" %}
```

Comment:
I see a newer version of the package here:

https://github.com/tylere/pykml/releases

Any reason not to use it?